change_logs/release_v0.17.7.md

```diff
@@ -14,7 +14,7 @@ On Slack? Please join us [K9slackers](https://join.slack.com/t/k9sers/shared_inv
 
 ## 🙀(PLUGIN-19)
 
-[BR]eaking bad on K9s plugins! In previous releases, we used the COL<INDEX> semantic to reference view column data in the plugin extensions. In this drop, we've axed this in favor of column name vs column index. This makes K9s plugin more readable and usable. Also, in light of custom columns, this old semantic just did not jive to well. To boot, all columns available on the viewed resource, regardless of display preferences or order are now free game to plugin authors. So for folks currently leveraging K9s plugins, this drop will break you I am hopeful you guys dig this approach betta??
```

Comment:
?

Reply:
As per the show. I guess should have been [Br]

Reply:
I think you want `[Br]eaking [Ba]d` -- totally happy to do this.

Amusingly, [betta](https://www.merriam-webster.com/dictionary/betta) here should really be `beta` (the former is a genus of 🐟 ).

Reply:
I went with `betta'` to match a similar instance. Although I'm going to have to consider adding `betta` to my list of "not words you want to use", since I suspect on average it's a typo of something.